src/main/java/frc/robot/subsystems/IntakeSystem.java

```diff
@@ -20,11 +20,11 @@ public IntakeSystem() {
     }
 
     public void out() {
-        motor.set(DEFAULT_ROTATE_SPEED);
+        motor.set(-DEFAULT_ROTATE_SPEED);
```

Comment:
why did you reverse the sign for `in` and `out`?

Reply:
turns out the positive speed is for intake and negative is for outtake